ingestion/functions/parsing/colombia/colombia_test.py

```diff
@@ -21,213 +21,493 @@ def test_parse(self):
+                           'administrativeAreaLevel1': 'BOGOTÁ, D.C.',
+                           'administrativeAreaLevel2': 'BOGOTÁ, D.C.'},
+              'demographics': {'ageRange': {'start': 19.0, 'end': 19.0},
+                               'gender': 'Female',
+                               'ethnicity': 'Other'},
+              'events': [{'name': 'confirmed',
+                          'dateRange': {'start': '03/06/2020Z', 'end': '03/06/2020Z'}},
+                         {'name': 'onsetSymptoms',
+                          'dateRange': {'start': '02/27/2020Z', 'end': '02/27/2020Z'}},
+                         {'name': 'outcome',
+                          'value': 'Recovered',
+                          'dateRange': {'start': '03/13/2020Z', 'end': '03/13/2020Z'}}],
+                'symptoms': {'status': 'Symptomatic', 'values': ['Mild']},
+                'travelHistory': {'traveledPrior30Days': True,
+                                  'travel': [{'location': {'country': 'Italy',
+                                                           'geoResolution': 'Country',
+                                                           'name': 'Italy',
+                                                           'geometry': {'latitude': 41.87194, 'longitude': 12.56738}}}]},
+                'notes': 'Patient self-isolated and recovered at home., Case is reported as importing the disease into Colombia, and country of origin is ITALIA., Date reported online was 03/06/2020., Date reported to SIVIGILA was 03/02/2020., Patient recovery was confirmed by a negative PCR test.'},
+                {'caseReference': {'sourceId': 'placeholder_ID',
+                                   'sourceEntryId': '2',
+                                   'sourceUrl': 'placeholder_URL'},
+                 'location': {'country': 'Colombia',
+                              'geoResolution': 'Admin2',
+                              'name': 'BUGA, VALLE DEL CAUCA, Colombia',
+                              'geometry': {'latitude': 3.85524552901928, 'longitude': -76.1051444045495},
+                              'administrativeAreaLevel1': 'VALLE DEL CAUCA',
+                              'administrativeAreaLevel2': 'BUGA'},
+                 'demographics': {'ageRange': {'start': 34.0, 'end': 34.0},
+                                  'gender': 'Male',
+                                  'ethnicity': 'Black'},
+                 'events': [{'name': 'confirmed',
+                             'dateRange': {'start': '03/09/2020Z', 'end': '03/09/2020Z'}},
+                            {'name': 'onsetSymptoms',
+                             'dateRange': {'start': '03/04/2020Z', 'end': '03/04/2020Z'}},
+                            {'name': 'outcome',
+                             'value': 'Recovered',
+                             'dateRange': {'start': '03/19/2020Z', 'end': '03/19/2020Z'}}],
+                 'symptoms': {'status': 'Symptomatic', 'values': ['Mild']},
```

Comment:
I thought that the symptoms.values field was meant to list actual symptoms according to the symptom ontology at https://docs.google.com/spreadsheets/d/1qOnGJQg1gtbmNxAW8c9hC1TvFpIQmSnORD-x-iXrpJ0/edit#gid=0. Maybe we should put symptom severity in notes?